tensorboard/components/tf_dashboard_common/tf-filterable-checkbox-list.html

```diff
@@ -60,14 +59,18 @@
         </div>
       </template>
       <template is="dom-repeat" items="[[_itemsMatchingRegex]]">
-        <div class="item">
+        <div class="item" id="[[item.id]]">
```

Comment:
Do we really want to assign the actual DOM element ID to `item.id` - isn't `item.id` potentially arbitrary?  It seems like it would open us up to ID clashes if somebody had an item with an ID of something like "sidebar".